hybrid/decomposers.py

```diff
@@ -828,6 +940,25 @@ def _make_cubic_lattice(dimensions):
                                         ])
     return cubic_lattice_graph
 
+def _make_kings_lattice(dimensions, is_open=(True,True)):
+    """Returns an open boundary cubic lattice graph as function of lattice
+    dimensions. Helper function for ``make_origin_embeddings``.
+    """
+    kings_lattice = nx.Graph()
```

Comment:
I like grid_graph, but you can't choose to make different boundary conditions for the two dimensions.
That is a very special use case, but I want to build it in since in the context of QPU embeddings its a restriction that is occasionally relevant.
That is probably a use case we don't need to cover, but at this stage don't want to rewrite.